### DISCUSSION TITLE
google serach console sitemap error

### DISCUSSION THREAD

**Discussion by 0321maki** [gatsbyjs/gatsby#28668] [Help]
We have an error of google search console sitemap
and the sitemap doesn't work. 
Please help to fix this!

![Screen Shot 2020-12-17 at 18 30 14](https://user-images.githubusercontent.com/76156378/102471073-19622080-4098-11eb-92ca-7429b8617828.png)
![Screen Shot 2020-12-17 at 18 47 43](https://user-images.githubusercontent.com/76156378/102471264-5c23f880-4098-11eb-802e-328b0b36df2a.png)

**Comment by vladar** [comment]
Hi @0321maki !

Do you have a `sitemap.xml` file in the `public` folder after the `gatsby build` command? If not, it is incredibly helpful if you're able to create a [minimal reproduction][reproduction]. This is a simplified example of the issue that makes it clear and obvious what the issue is and how we can begin to debug it.

If you're up for it, we'd very much appreciate if you could [provide a minimal reproduction][reproduction] (or at least an access to your code) and we'll be able to take another look.

Thanks for using Gatsby! 💜

[reproduction]: https://gatsby.dev/reproduction

**Reply by 0321maki** [reply]
Thank you for your reply. 
I will try!!
